lib/main_common.pm

```diff
@@ -346,7 +346,16 @@ sub load_yast2_ui_tests {
     loadtest "console/hostname";
     loadtest "console/zypper_lr";
     loadtest "console/zypper_ref";
-    # start extra yast console test from here
+    # split YaST2 UI tests relying on external development controlled test
+    # suites and self-contained ones
+    if (get_var('Y2UITEST_DEVEL')) {
+        # (Livesystem and laptops do use networkmanager)
+        if (!get_var("LIVETEST") && !get_var("LAPTOP")) {
+            loadtest 'console/yast2_cmdline';
+        }
+        return;
+    }
```

Comment:
no, see above. If `Y2UITEST_DEVEL` is set there is no need to also schedule all other modules.